hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/token/queries/GetTokenInfoResourceUsage.java

```diff
@@ -96,6 +96,8 @@ public FeeData usageGiven(final Query query, final Token token) {
                     .givenCurrentKycKey(token.hasKycKey() ? Optional.of(fromPbj(token.kycKey())) : Optional.empty())
                     .givenCurrentPauseKey(
                             token.hasPauseKey() ? Optional.of(fromPbj(token.pauseKey())) : Optional.empty())
+                    .givenCurrentPauseKey(
```

Comment:
⚠️ This looks like a typo; should it it be deleted?

Reply:
Should be `givenCurrentMetadataKey`, I will change it. Thanks @tinker-michaelj

Reply:
Fixed. Thanks @tinker-michaelj